src/index.html

```diff
@@ -10,13 +10,59 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo-image"
+        />
+      </div>
+      <nav class="nav">
+        <ul class="menu">
+          <li class="menu-item">
+            <a
+              class="is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu-item"><a href="#">Samsung</a></li>
+          <li class="menu-item"><a href="#">Smartphones</a></li>
+          <li class="menu-item">
+            <a
+              href="#"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="menu-item"><a href="#">Gadgets</a></li>
+          <li class="menu-item"><a href="#">Tablets</a></li>
+          <li class="menu-item"><a href="#">Photo</a></li>
+          <li class="menu-item"><a href="#">Video</a></li>
```

Comment:
Similar to the previous comment, all href attributes are set to '#'. Ensure that these are placeholders and will be updated with actual URLs in the future.